projects/vdk-plugins/vdk-ipython/setup.py

```diff
@@ -14,7 +14,7 @@
     description="Ipython extension for VDK",
     long_description=pathlib.Path("README.md").read_text(),
     long_description_content_type="text/markdown",
-    install_requires=["vdk-core", "iPython"],
+    install_requires=["vdk-core", "iPython", "pandas"],
```

Comment:
Same question here.

Reply:
I made pandas a required dependency here because the likelihood of not having a pandas in a notebook installation is extremely low. Pandas is a de-facto standard for python based transformations. 

On the other hand, ippygrid is a specific widget that may not work across all versions of jupyter or other notebook solutions. I certainly haven't tested it. And it's better if it's optional